### ISSUE TITLE
Does it compatible with Chinese pre-trained model?

### ISSUE THREAD

**Issue by Flyfoxs** [CyberZHG/keras-xlnet#13]
I try it, and  not exception. but the accuracy is very low. So not sure it support Chinese pretrained mode or not.
https://github.com/ymcui/Chinese-PreTrained-XLNet

**Comment by lsq357** [comment]
@Flyfoxs   Which dataset and how encode text to spiece ?

**Comment by Flyfoxs** [comment]
use the dict in this link, the  dict is  include in the pretrain model   https://github.com/ymcui/Chinese-PreTrained-XLNet

you can find the pretrain mode in this link: https://drive.google.com/open?id=1342uBc7ZmQwV6Hm6eUIN_OnBSz1LcvfA

**Comment by CyberZHG** [comment]
@Flyfoxs

The Chinese version uses a different activation function for feed-forward layers. You can upgrade to `0.17.0` and the outputs will be the same in this [demo](https://github.com/CyberZHG/keras-xlnet/blob/master/demo/extract/token_embed_chinese_mid.py).

**Comment by Flyfoxs** [comment]
thanks . so much
